clustered/client/src/main/java/org/ehcache/clustered/config/ClusteringServiceConfiguration.java

```diff
@@ -58,4 +87,44 @@ public URI getConnectionUrl() {
   public CacheManagerBuilder<PersistentCacheManager> builder(final CacheManagerBuilder<? extends CacheManager> other) {
     return (CacheManagerBuilder<PersistentCacheManager>) other.using(this);
   }
+
+  public static final class PoolDefinition {
+
+    private final long size;
+    private final MemoryUnit unit;
+    private final String from;
+
+    public PoolDefinition(long size, MemoryUnit unit, String from) {
```

Comment:
I need to add javadoc to this... and a bunch of other places... let me do that now.